engine/src/core/utils/file_manager.cpp

```diff
@@ -8,8 +8,11 @@
 
 #ifdef LINUX
     #include <sys/mman.h>
+    #include <sys/stat.h>
+    #include <fcntl.h>
```

Comment:
What are these?

Reply:
Some system libraries I had to add them because of your big file manager :P.